app/Actions/FarmerReport/CreateFarmerReport.php

```diff
@@ -32,6 +33,36 @@ public function asOrchidAction($model, ?Request $request)
         return redirect()->route('platform.farmer-reports');
     }
 
+    //PROXYYYY PALANG TOH HAHAHAH di ko pa alam ano ilalagay hehehe
```

Comment:
Make the Swagger documentation more appropriate. Change the values inside each property. For example, define the required parameters to create a farmer report in the `@OA\JsonContent` part of the comment.

Reply:
Updated the Swagger documentation.

Reply:
CI is not passing. Check the error log [here](https://github.com/Radical-Doubting/anikultura-backend/runs/4997322562?check_suite_focus=true).

Reply:
Fixed and updated the issure regarding multiple @OA\Property.